act/codereview.vars

```diff
@@ -0,0 +1,3 @@
+OPENAI_BASE_URL=https://cds-snc.openai.azure.com/openai/deployments/gpt-4o/
+OPENAI_API_MODEL=gpt-4o
+OPENAI_API_VERSION=2024-02-15-preview
```

Comment:
A `.secrets` file can sit beside this file to use [act](https://github.com/nektos/act/) and locally debug this Github action. It should contain both a github token and the Azure OpenAI API token.